kaggle_environments/envs/halite/halite.py

```diff
@@ -87,9 +88,11 @@ def populate_board(state, env):
     uid_counter = 0
 
     # Set seed for random number generators
-    if hasattr(config, "randomSeed"):
-        np.random.seed(config.randomSeed)
-        seed(config.randomSeed)
+    if not hasattr(config, "randomSeed"):
+        config.randomSeed = randrange(sys.maxsize)
```

Comment:
This will export seeds for halite episodes so users can reproduce their production episodes.

Reply:
sys.maxsize is 2**63-1 on 64-bit systems, it does not work with np.random.seed which only handles seeds 0 .. 2**32-1.

I don't think that users can exactly reproduce their production episodes, unless they explicitly fork off their own exclusive random number generator (and the seed for that one can be logged using the convential logging api).

Reply:
Thanks, I haven't run this PR through our test environment yet so it's totally possible I missed other issues like this.  
  
I'm not sure I understand your point about reproducibility -- randomSeed is passed into agents as well through the configuration parameter so agents should be able to consume this field to seed their own rngs.